<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListApp</class>
 <widget class="QMainWindow" name="ListApp">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ListApp</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton{
border: 2px solid blac;
border-radius: 8px;
background-color: qlineargradient(spread:pad, x1:0.542, y1:1, x2:0.543, y2:0, stop:0 rgba(108, 74, 74, 255), stop:1 rgba(255, 255, 255, 255));
}

.QPushButton:hover{
background-color:rgb(127, 96, 99);
color:white;
border: 2px solid black;
}

QWidget#acceuilWidget{
background-color:rgb(214, 214, 214);
border: 2px solid black;
border-radius: 8px;
}</string>
    </property>
    <widget class="QWidget" name="accueilPage">
     <widget class="QWidget" name="acceuilWidget" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="accueilLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="accueilLabel">
          <property name="font">
           <font>
            <family>Suruma</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="text">
           <string>List App</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aJouterBouton">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="visualiserBouton">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Visualiser</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="visualiserPage">
     <widget class="QScrollArea" name="logsScroll">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>611</width>
        <height>451</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="logsScrollContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>609</width>
         <height>449</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="logsContent">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>611</width>
          <height>381</height>
         </rect>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>391</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="logsHeaderLayout">
         <item>
          <widget class="QLabel" name="headerId">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="headerNom">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="headerPrenom">
           <property name="text">
            <string>Prenom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="headerAge">
           <property name="text">
            <string>Age</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="RetourBouton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>502</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouterPage">
     <property name="styleSheet">
      <string notr="true">.QLineEdit{
margin-right: 20px;
margin-left: 20px;
border:none;
border-right: 2px solid black;
border-bottom: 2px solid black;
height:20px
}

.QSpinBox{
margin-right: 20px;
margin-left: 20px;
border:none;
border-right: 2px solid black;
border-bottom: 2px solid black;
}


.QTextEdit:clicked {
background-color:rgb(241, 241, 241);
border-right:2px solid green;
border-bottom:2px solid green;
}

.QLabel#ajouterLabel{
text-decoration:underline;
}

.QLabel#errorNom, .QLabel#errorAge, .QLabel#errorPrenom{
color:red;
}</string>
     </property>
     <widget class="QPushButton" name="RetourBoutonAjout">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>441</width>
        <height>360</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ajouterLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="ajouterLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>22</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Ajouter une Nouvelle Entr√©e</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="ajouterForm">
         <item row="0" column="0">
          <widget class="QLabel" name="nomLabel">
           <property name="text">
            <string>Nom:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nomText">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="errorNom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Nom invalide</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="prenomLabel">
           <property name="text">
            <string>Prenom: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="prenomText">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="errorPrenom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Prenom invalide</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ageLabel">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Age: </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="ageText">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="errorAge">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Age invalide</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="validerBouton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorWidget" native="true">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>220</y>
        <width>171</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>160</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="errorLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="testPage">
     <widget class="QScrollArea" name="testScroll">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>721</width>
        <height>461</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="testContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>719</width>
         <height>459</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
     <widget class="QPushButton" name="RetourBoutonTest">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
